### DISCUSSION TITLE
OnlyFans credentials expired

### DISCUSSION THREAD

**Discussion by XinSol** [AAndyProgram/SCrawler#176] [Q&A] [answered]
form the log:
2023-12-28 10:15:56: [OnlyFans] - elexxxxxxx [400]: OnlyFans credentials expired

I reconfigured de with new cookies and codes

Why?

**Comment by AAndyProgram** [comment, marked as answer]
**Join our Discord server**: https://discord.gg/uFNUXvFFmg

![image](https://github.com/AAndyProgram/SCrawler/assets/88590076/5ff5125f-91d5-4630-b270-86f769165b82)

**Reply by CynicalPlatapus** [reply]
cheers for this, was doing my fucking nut in trying to get it working again, might be worth adding this to the guide

**Reply by AAndyProgram** [reply]
**Join our Discord server**: https://discord.gg/uFNUXvFFmg

**Reply by AAndyProgram** [reply]
https://github.com/AAndyProgram/SCrawler/issues/177

**Reply by k7bscs4ttf-collab** [reply]
-
